### DISCUSSION TITLE
Querynode can't comunicate with GCS cloud

### DISCUSSION THREAD

**Discussion by Trourest186** [milvus-io/milvus#36515] [Q&A and General discussion]
Hi everyone,

Recently, i deployed a group of nodepoods to GKE to serve my projects. I deployed milvus with helmchart (image: milvusdb/milvus:v2.4.1) to there. But, only querynode can't have running status. Checking logs in pod, it raised errors below, who can suggest solution for this problem. Thanks so much!!!

Log of querynodes
![image](https://github.com/user-attachments/assets/dc537502-5662-4160-a943-d2a7d5ab2651)

Get for all milvus' node
![image](https://github.com/user-attachments/assets/0180c802-e28b-4d75-ab37-60e559fae5c1)

Best regards!
Giang

**Comment by yhmo** [comment]
Seems the query node could not access the gcs. Wait @LoveEachDay to comment.

**Reply by LoveEachDay** [reply]
@Trourest186 Could you show us the helm chart values.yaml? Please remove the senstive information. By the way, you can use `mc` binary to verify the gcs bucket access.

Learn more about mc from https://min.io/docs/minio/linux/reference/minio-mc.html

**Reply by Trourest186** [reply]
> @Trourest186 Could you show us the helm chart values.yaml? Please remove the senstive information. By the way, you can use `mc` binary to verify the gcs bucket access.
> 
> Learn more about mc from https://min.io/docs/minio/linux/reference/minio-mc.html

Thanks you, i send you my values.yaml file, hope you spend time to check and support. 
[values.yaml.txt](https://github.com/user-attachments/files/17145849/values.yaml.txt)

**Reply by LoveEachDay** [reply]
@Trourest186 The `values.yaml` attached above is solid. Could you use `mc` to verify the `accessKey` and `secretKey` for you gcs bucket is valid?

**Reply by Trourest186** [reply]
> @Trourest186 The `values.yaml` attached above is solid. Could you use `mc` to verify the `accessKey` and `secretKey` for you gcs bucket is valid?

I used service account have "Storage Admin" role, and create access key from HMAC in cloud storage such as image, then use "mc" command to setup alias, and mc admin to test connection. But it appears errors
![image](https://github.com/user-attachments/assets/6c075920-5ff7-4cca-83c8-0a215533e308)
![mc](https://github.com/user-attachments/assets/b0ce0a65-2d88-446e-b7cb-3a390af6b347)
